tests/helper/helper_interactive.go

```diff
@@ -0,0 +1,66 @@
+// +build linux darwin dragonfly solaris openbsd netbsd freebsd
+
+package helper
+
+import (
+	"bytes"
+	"log"
+	"os/exec"
+
+	"github.com/Netflix/go-expect"
+	"github.com/hinshun/vt10x"
+	"github.com/kr/pty"
+	. "github.com/onsi/gomega"
+)
+
+func RunInteractive(commonVar CommonVar, command []string, test func(*expect.Console, *bytes.Buffer)) (string, error) {
```

Comment:
Please add a comment as to what `RunInteractive` does and what it does with the parameters it takes.

Reply:
Comment added, and removed `commonVar`